http-plane-headers-extension/build.gradle.kts

```diff
@@ -0,0 +1,136 @@
+plugins {
+    `java-library`
+    id("application")
+    `maven-publish`
+    signing
+    alias(libs.plugins.shadow)
+}
+
+group = "org.eclipse.edc"
```

Comment:
Group id is still set to eclipse?

Reply:
If you set the top level group to `be.vlaanderen.informatievlaanderen` you get the following error:
`Error: Could not find or load main class be.vlaanderen.informatievlaanderen.boot.system.runtime.BaseRuntime`

Only when set to `org.eclipse.edc` the application works as intended.